docs/guides/state-sync.md

```diff
@@ -137,40 +168,40 @@ increase then do the following:
 1. Double-check the node configuration file has correctly been updated.
 2. Make sure that the node is starting up with an empty storage database
    (i.e., that it has not synced any state previously).
+   :::
 
-## Running archival nodes
+## Running archival PFNs
 
-To operate an archival node, which is a fullnode that contains all blockchain data
-since the start of the blockchain's history (that is, genesis), you should:
+To operate an archival PFN (which is a PFN that contains all blockchain data
+since the start of the network, i.e., genesis), you should:
 
-1. Run a fullnode and configure it to either: (i) execute all transactions; (ii) apply all transaction outputs; or (iii)
-   use intelligent syncing (see above). Do not select fast syncing, as the fullnode will not contain all data since genesis.
+1. Configure the PFN _not to use_ fast syncing, e.g., select intelligent syncing in your node configuration file.
```

Comment:
Does it make sense to automatically disable pruning when intelligent syncing is enabled, and disable it otherwise ?

Reply:
No, they are orthogonal. Intelligent syncing works with and without pruning. Pruning is only if folks want archival nodes 😄